.gitignore

```diff
@@ -26,70 +27,70 @@ dependency-reduced-pom.xml
 .idea/kubernetes-settings.xml
 
  # AWS User-specific
- .idea/**/aws.xml
+.idea/**/aws.xml
 
  # Generated files
- .idea/**/contentModel.xml
+.idea/**/contentModel.xml
 
  # Sensitive or high-churn files
- .idea/**/dataSources/
- .idea/**/dataSources.ids
- .idea/**/dataSources.local.xml
- .idea/**/sqlDataSources.xml
- .idea/**/dynamic.xml
- .idea/**/uiDesigner.xml
- .idea/**/dbnavigator.xml
+.idea/**/dataSources/
+.idea/**/dataSources.ids
+.idea/**/dataSources.local.xml
+.idea/**/sqlDataSources.xml
+.idea/**/dynamic.xml
+.idea/**/uiDesigner.xml
+.idea/**/dbnavigator.xml
 
  # Gradle
- .idea/**/gradle.xml
- .idea/**/libraries
-
- # Gradle and Maven with auto-import
- # When using Gradle or Maven with auto-import, you should exclude module files,
- # since they will be recreated, and may cause churn.  Uncomment if using
- # auto-import.
- # .idea/artifacts
- .idea/compiler.xml
- .idea/encodings.xml
- # .idea/jarRepositories.xml
- # .idea/modules.xml
- # .idea/*.iml
- # .idea/modules
- # *.iml
- # *.ipr
+.idea/**/gradle.xml
+.idea/**/libraries
+
+# Gradle and Maven with auto-import
+# When using Gradle or Maven with auto-import, you should exclude module files,
+# since they will be recreated, and may cause churn.  Uncomment if using
+# auto-import.
+# .idea/artifacts
```

Comment:
Should we un-comment the rest of these as well at the same time?

Reply:
sounds good